packages/editor/src/components/document-outline/item.js

```diff
@@ -49,8 +46,7 @@ const TableOfContentsItem = ( {
 			<span className="document-outline__item-content">
 				{ children }
 			</span>
-			<span className="screen-reader-text">{ __( '(Click to focus this heading)' ) }</span>
```

Comment:
Why did you remove the screen reader text?

Reply:
hey @IreneStr - i think the screen reader text is no longer needed because screen readers properly announce the a href tags so they are no longer needed? @afercia suggested removing in this comment: https://github.com/WordPress/gutenberg/pull/10815#issuecomment-437581453. do you think it is still needed? i have tested with voiceover and the announcements seemed sufficient, however I'm not a regular screen reader so i'm a little unsure what to expect.

Reply:
Good to know. I retested with the screen reader and I'm convinced that what's announced is sufficient. I was asking because `Click to focus this heading` was explicitly mentioned in the HTML in the original issue. 

Let's get this merged!